releasenotes/notes/own_assembler-0c76e67a054bd12c.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Aer simulates :class:`~qiskit.circuit.QuantumCircuit` without serialization to
+    :class:`~qiskit.qobj.QasmQobj`. Instead, Aer converts a circuit to
+    a :class:`.AerCircuit` instance that a wrapper of internal circuit structure
+    of Aer C++. With this change, because overheads of serialization is reduced,
+    performance of simulation especially for low-qubit circuts is improved.
+    Note that pulse simualation and DASK-based simulation still use Qobj and
+    :class`.AerJob` does not return Qobj otherwise.
```

Comment:
We probably should have an additional note in this file for the addition of `AerJob.circuits` since that's also a new API.